### PR TITLE
mobile: more spanish

### PR DESCRIPTION
**Pull request by dcposch**
Screenshot 2024-07-27 at 02 23 52
Screenshot 2024-07-27 at 03 37 49
Screenshot 2024-07-27 at 03 37 56
Screenshot 2024-07-27 at 03 38 09
Screenshot 2024-07-27 at 03 38 12
Screenshot 2024-07-27 at 03 38 19
Screenshot 2024-07-27 at 03 38 34
Screenshot 2024-07-27 at 02 23 58
Screenshot 2024-07-27 at 02 24 01
Screenshot 2024-07-27 at 02 24 09
Screenshot 2024-07-27 at 02 24 21
Screenshot 2024-07-27 at 03 40 23
Screenshot 2024-07-27 at 02 26 13
Screenshot 2024-07-27 at 02 27 14
Screenshot 2024-07-27 at 02 37 08
Screenshot 2024-07-27 at 02 37 14
Screenshot 2024-07-27 at 02 37 23
Screenshot 2024-07-27 at 02 37 27
Screenshot 2024-07-27 at 02 37 30
Screenshot 2024-07-27 at 02 37 38
Screenshot 2024-07-27 at 02 37 48
Screenshot 2024-07-27 at 02 38 06
Screenshot 2024-07-27 at 02 38 27
Screenshot 2024-07-27 at 02 38 40
Screenshot 2024-07-27 at 02 38 45
Screenshot 2024-07-27 at 02 38 51
Screenshot 2024-07-27 at 02 43 23
Screenshot 2024-07-27 at 02 43 41
Screenshot 2024-07-27 at 02 43 54
Screenshot 2024-07-27 at 02 43 57
Screenshot 2024-07-27 at 02 44 04
Screenshot 2024-07-27 at 02 44 15
Screenshot 2024-07-27 at 02 48 05
Screenshot 2024-07-27 at 02 53 08
Screenshot 2024-07-27 at 02 53 32
Screenshot 2024-07-27 at 02 54 20
Screenshot 2024-07-27 at 02 54 31
Screenshot 2024-07-27 at 03 10 03
Screenshot 2024-07-27 at 03 10 05
Screenshot 2024-07-27 at 03 14 28

### CODE REVIEW

**Comment by Ethanol48** [comment]
I'm seeing some strings that are a bit strange, I'll have a look this afternoon.

**Comment by Ethanol48** [comment]
I'll create a PR for simplicity

**Comment by Ethanol48** [comment]
I found some strings that need to be exported to i18n

**Comment by Ethanol48** [comment]
I found another bug, in the simulator, in the `Deposit` tab the line for deposits doesn't seem to follow the translation, I've check the `es.ts` file and is not a typo, 

For Withdraw this doesn't happen, it works fine, I have look through the code, but I couldn't find the source.

attaching a screenshot for showcase:

Screenshot 2024-07-27 at 14 46 06

**Comment by Ethanol48** [comment]
I also encountered some hardcoded text in some titles of Modals imported to `DepositScreen.tsx`, found inside the `sheet` folder. I'm not sure if this text is critical for functionality, I wanted to point it out.

**Comment by Ethanol48** [comment]
I found a bug, seems to be from the package "daimo-common", it returns the english translation when it should be the spanish, I've check and the translation is spanish is set correctly.

Screenshot 2024-07-27 at 15 08 16

it should be `hace {time}`

File: apps/daimo-mobile/src/view/shared/SearchResults.tsx
Comment (Ethanol48) [inline apps/daimo-mobile/src/view/shared/SearchResults.tsx:1]: In line 93 of this file, there's some texts that need to be refactored for i18n

**Review by Ethanol48 — CHANGES_REQUESTED** [review]
Mostly refactoring for i18n

**Comment by Ethanol48** [comment]
Sorry about the request changes, I'm using my phone and the interface isn't the greatest in the app, there are some requests that are duplicated.

**Comment by Ethanol48** [comment]
Added some fields in i18n and implemented them where it was needed, made some final touches to the spanish translations, it should be good to go for the daimo-mobile part.

I'm still having some issues from "daimo-common" that is giving the english translation when it should be the spanish.

This been resolved, it should be good to go.

**Comment by dcposch** [comment]
@Ethanol48 your updates look great, thanks!

**Pull request merged**